lethal-squirrels/workforce-management
# Ben db seeder

## DESCRIPTION
Made a DB initializer so that we have seed data to work with.
## STEPS TO TEST
1. Clone this branch to a local repo
2. In the package manager, run `update-database`. 
3. Test functionality
	* Check that the data exists when you go to localhost:####/employees, computers etc.
